target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -42,6 +42,10 @@ ramips_setup_interfaces()
 	mikrotik,rb750gr3)
 		ucidef_set_interfaces_lan_wan "lan2 lan3 lan4 lan5" "wan"
 		;;
+	zyxel,wap6805)
```

Comment:
Please move to follow alphabetic sorting.

Reply:
This file is not sorted.  But moving z last is probably better in any case.  Fixed

Thanks a lot for your review and very useful feedback.  I have made all the requested changes, and rebuilt and booted the result to try to make sure I didn't mess up anything..

Reply:
> This file is not sorted. But moving z last is probably better in any case. Fixed

It actually is sorted, blocks are arranged based on their first devices, and for one block the devices are again sorted alphabetically. At least it should be this way.

> Thanks a lot for your review and very useful feedback. I have made all the requested changes, and rebuilt and booted the result to try to make sure I didn't mess up anything..

Well, you're welcome, it's just cosmetic nitpicking anyway.